spec.md

```diff
@@ -1977,14 +1977,15 @@ However it can never be used in a reference since it is not of any valid type.
 
 The assert statement verifies that the predicate is true on the rising edge of any clock cycle when the enable is true.
 In other words, it verifies that enable implies predicate.
+When the predicate is false, the assert statement prints out the format string where its argument placeholders are substituted.
```

Comment:
L1959 describes the operands of all verification statements and refers to this as "string literal" and does not mention the new additional operands, please align.

(it does mention the message (without format-string behavior naturally) may be printed for the assert/assume cases already, FYI -- but address however you see fit just please keep these sections on same page please).

The language currently is guarded by "may".